Tests/Tests.csproj

```diff
@@ -181,7 +169,6 @@
   <ItemGroup>
     <Content Include="Data\CKAN.sln" />
     <Content Include="Data\cmdline.t" />
-    <Content Include="Data\data\CKAN.sln" />
     <Content Include="Data\data\KSP\bad-KSP\readme.txt" />
     <Content Include="Data\data\KSP\KSP-0.25\readme.txt" />
     <Content Include="Data\data\KSP\missing-gamedata\readme.txt" />
```

Comment:
Can probably get rid of these three `Data\data` files as well?

Reply:
Happy to, if they aren't used.